pkg/web/static/js/transactionAcceptPreview.js

```diff
@@ -1,4 +1,4 @@
-import { networksArray, legalPersonNameTypeArray, addressTypeArray, nationalIdentifierTypeArray, naturalPersonNameTypeArray } from "./constants.js";
+import { countriesArray, networksArray, legalPersonNameTypeArray, addressTypeArray, nationalIdentifierTypeArray, naturalPersonNameTypeArray } from "./constants.js";
```

Comment:
This resolves an error as `countriesArray` wasn't imported from the `constant.js` file before being used.